apps/studio/stores/pgmeta/MetaStore.ts

```diff
@@ -426,7 +426,8 @@ export default class MetaStore implements IMetaStore {
     const connectionString = project?.connectionString
     const tables = await queryClient.fetchQuery({
       queryKey: tableKeys.list(projectRef, 'public'),
```

Comment:
I'm not sure if this needs changing too?
```suggestion
      queryKey: tableKeys.list(projectRef, sourceTableSchema),
```